File: server/src/main/java/com/example/schoolsite/pojo/SignUpRequest.java
package com.example.schoolsite.pojo;

import lombok.Getter;
import lombok.Setter;

import java.util.Set;

@Getter
@Setter
public class SignUpRequest {
    private String name;
    private String lastname;
    private String patronymic;
    private String email;
    private String login;
    private String password;
    private Set<String> role;
    private String status;
    private String link;
}

File: server/src/main/java/com/example/schoolsite/pojo/SignUpRequest.kt
package com.example.schoolsite.pojo

import lombok.Getter
import lombok.Setter
import java.util.Set

@Getter
@Setter
class SignUpRequest {
    private val name: String? = null
    private val lastname: String? = null
    private val patronymic: String? = null
    private val email: String? = null
    private val login: String? = null
    private val password: String? = null
    private val role: Set<String>? = null
    private val status: String? = null
    private val link: String? = null
}